test/cli/launch.test.js

```diff
@@ -8,7 +8,9 @@ const startCLI = require('./start-cli');
 test('examples/empty.js', (t) => {
   const script = Path.join('examples', 'empty.js');
   const cli = startCLI([script]);
-  return cli.waitForPrompt()
+
+  return cli.waitFor(/break/)
```

Comment:
This matches how all other test files are doing it. There really should be a `cli.waitForInitialPrompt` helper.